### PR TITLE
feat(cli): use file services resolver in build stylable

### PR DESCRIPTION
**Pull request by tzachbon**
Has mentioned [here](https://github.com/wixplosives/file-services/issues/24#issuecomment-1014338289), this gives us a boost in terms of performance which we like in our CLI for big projects that are going to use our [integrations (e.g., webpack, rollup) with `stc`](https://github.com/wix/stylable/pull/2342).
The downside is that the resolver has missing features, but they are very advanced, and most modern tools still do not support them as well.

### CODE REVIEW

File: packages/cli/src/resolve-module.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import type { IFileSystem } from '@file-services/types';
+import { createRequestResolver } from '@file-services/resolve';
+import type { StylableConfig } from '@stylable/core';
+
+export function createDefaultResolveModule(fs: IFileSystem): StylableConfig['resolveModule'] {
```
Comment (AviVahl) [inline packages/cli/src/resolve-module.ts:5]: This is a much larger type than the resolver needs/uses. See it's options. There's another, smaller interface.

File: packages/cli/src/resolve-module.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    const moduleResolver = createRequestResolver({ fs });
+
+    return (context, request) => {
+        const { resolvedFile } = moduleResolver(context, request);
+
+        if (resolvedFile === false) {
+            throw new Error(
+                `Stylable CLI does not support browser field 'false' values. "${request}" resolved to 'false' from "${context}"`
```
Comment (AviVahl) [inline packages/cli/src/resolve-module.ts:13]: How is stylable core handling it?

Comment (AviVahl) [reply packages/cli/src/resolve-module.ts:13]: nvmind https://github.com/wix/stylable/blob/master/packages/core/src/module-resolver.ts#L38

**Comment by tomrav** [comment]
Closing for now (keeping branch).

**Pull request closed without merging**